### DISCUSSION TITLE
A running virtual device freezes after resuming from hibernate

### DISCUSSION THREAD

**Discussion by hope2586** [Genymobile/feedback#39] [Genymotion Desktop] [answered]
It would be nice if it was fixed since it doesn't happen in BlueStacks.

**Comment by agavignet** [comment]
@hope2586 what is your operating system so we can test on our side? thank you

**Reply by hope2586** [reply]
Windows 10. It seems to be an issue with VirtualBox saving the state of the Virtual Device and not resuming it when the host OS is resumed from hibernation. I tried switching to QEMU and installing HAXM in order to see if the issue happens with it, but it doesn't boot.

**Reply by hope2586** [reply]
Any news?

**Reply by jmaneyrol69** [reply, marked as answer]
Hi @hope2586,

Sorry for the late answer.
Since your last post, a new version of Desktop has been released (3.3.2) which is now compatible with VirtualBox 7.0.4.
Do you still have this issue with this update and using VBox 7.0.4?
